### PR TITLE
Updates to packaging/deployment markdown files

### PR DESCRIPTION
**Pull request by DavidBoike**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -1,24 +1,39 @@
 # Deployment
 
-All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md).
+All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md). During packaging, deployment binaries are created in the `deploy` directory, some of which are packaged into zip files in the `zips` folder, and other deployment artifacts are built from these pieces.
 
 ## ServiceControl installer
 
-A [windows installer is provided for download](https://docs.particular.net/servicecontrol/installation) containing both ServiceControl Management Utility (SCMU) and [Powershell commands](https://docs.particular.net/servicecontrol/powershell).
+The zips are packaged as embedded resources in the ServiceControl Management Utility (the "installer").
```
Comment (bording) [inline docs/deployment.md:7]: As mentioned in the packaging doc, they are actually embedded in the installer assembly, not directly in the SCMU exe. Does this matter for this documentation? Not sure,

File: docs/deployment.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
-All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md).
+All ServiceControl deployment options listed below rely on the files produced by [packaging](packaging.md). During packaging, deployment binaries are created in the `deploy` directory, some of which are packaged into zip files in the `zips` folder, and other deployment artifacts are built from these pieces.
 
 ## ServiceControl installer
 
-A [windows installer is provided for download](https://docs.particular.net/servicecontrol/installation) containing both ServiceControl Management Utility (SCMU) and [Powershell commands](https://docs.particular.net/servicecontrol/powershell).
+The zips are packaged as embedded resources in the ServiceControl Management Utility (the "installer").
 
-When installing an instance using one of the above method instance files, persister (if applicable) and transport files are unzipped into the same folder.
+When installing an instance using one of the above method instance files, the following binaries are unzipped together to form a complete instance:
```
Comment (bording) [inline docs/deployment.md:9]: > When installing an instance using one of the above method instance files

Huh?

File: docs/deployment.md [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
+
+- Application & persistence files
+- Selected transport
+- RavenDB server files (ServiceControl and Audit only)
```
Comment (bording) [inline docs/deployment.md:13]: This list of files doesn't seem to line up with what actually gets unzipped.

For example, all transports are on disk, not just the one selected.

File: docs/deployment.md [hunk 4]
```diff
@@ -1,24 +1,39 @@
...
 
 A configuration file generated based on:
 
 - User input (and defaults)
-- Contents of the `persister.manifest` file of the selected persister
-
-## NuGet package
+- Contents of the `transport.manifest` and `persister.manifest` file for the selected transport and persister.
```
Comment (bording) [inline docs/deployment.md:18]: I know a version of this line already existed, but in what way are the contents of the manifest files used to create a config file?

File: docs/packaging.md [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
-- `/Transports` contains a folder for each supported transport
-- `/Persisters` contains a folder for each supported persister (Only the Audit instance use this currently)
-- `/<product>` contains the instance files specific to the type of instance
-  - `/ServiceControl`
-  - `/ServiceControl.Audit`
-  - `/ServiceControl.Monitoring`
+- `ServiceControl.zip`
+- `ServiceControl.Audit.zip`
+- `ServiceControl.Monitoring.zip`
+- `InstanceShared.zip` - Contains the transport assemblies that are shared by each instance
```
Comment (bording) [inline docs/packaging.md:10]: This will contain any files that are shared across all 3 instance zips, not just transports.

**Review by bording — CHANGES_REQUESTED** [review]

File: docs/deployment.md (hunk 1 above)
Comment (DavidBoike) [reply docs/deployment.md:7]: Yeah just meant to mean "In there somewhere" and I can't think of a better way to say it without repeating everything in packaging.md.

File: docs/deployment.md (hunk 3 above)
Comment (DavidBoike) [reply docs/deployment.md:14]: Added a couple lines, should be closer to the full truth now.

File: docs/deployment.md (hunk 1 above)
Comment (bording) [reply docs/deployment.md:7]: Maybe this could just say that SCMU is self-contained single exe that has everything inside it?

Comment (DavidBoike) [reply docs/deployment.md:7]: This good?
```suggestion
The zips are packaged as embedded resources in the ServiceControl Management Utility. Although technicall embedded in a specific assembly, ServiceControl Management is shipped as a self-contained, single-file executable that has everything inside it.
```

**Review by bording — APPROVED** [review]

**Pull request merged**